requirements.txt

```diff
@@ -6,7 +6,7 @@ Flask==1.1.2
 lxml>=4.2.5
 Pillow
 requests>=2.22.0,<3
-scikit-learn==0.24.1
+scikit-learn>=0.24.1
```

Comment:
Could we use `~`  instead of `>=` ? It's risky to keep all version there.

Reply:
@melihozaydin I've checked ML backend codes and haven't found `import sklearn` anywhere.. so, let's try to remove it completely and see what will happen with it?

Reply:
Done.